resources/ec2-internet-gateway-attachments.go

```diff
@@ -81,6 +83,7 @@ func (e *EC2InternetGatewayAttachment) Properties() types.Properties {
 	for _, tagValue := range e.vpcTags {
 		properties.SetTagWithPrefix("vpc", tagValue.Key, tagValue.Value)
 	}
+	properties.Set("DefaultVpcAssoc", e.defVpcAssoc)
```

Comment:
I would simply strip the `Assoc` part here. Also to follow convention `VPC` should be spelled uppercase.

```suggestion
	properties.Set("DefaultVPC", e.defaultVPC)
```